.rubocop.yml

```diff
@@ -11,7 +11,7 @@ Rails/ApplicationRecord:
   Enabled: false
 
 LineLength:
-  Max: 99
+  Max: 160
```

Comment:
:thinking: i prefer to keep max line length in 99

Reply:
Done.
I was trying to avoid one of Rubocop's warnings.
I had to modify BlockLength.